README.md

```diff
@@ -110,7 +110,8 @@ Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Go
 Specification: General|Every Tuesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1-bCYkN-DWJq4jw1ybaDZYYmx-WAe6HnwfWbkm8d57v8/edit?usp=sharing)|[Zoom](https://zoom.us/j/6729396170)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-specification)
 Specification: Errors|Every Thursday at 11:00 PT|[Google Doc](https://docs.google.com/document/d/1HgUI69rBridFzCXxXuTjQrkG6jb-YcFQnZjPcyBcK1U/edit#)|[Zoom](https://zoom.us/j/6729396170)|[Gitter](https://gitter.im/open-telemetry/error-events-wg)|
 Specification: Logs|Every other week on Wednesday at 10:00 PT|[Google Doc](https://docs.google.com/document/d/1cX5fWXyWqVVzYHSFUymYUfWxUK5hT97gc23w595LmdM/edit#)|[Zoom](https://zoom.us/j/8203130519)|[Gitter](https://gitter.im/open-telemetry/logs)
-Specification: Metrics|Every Thursday at 11:00 PT|[ Google Doc](https://docs.google.com/document/d/1LfDVyBJlIewwm3a0JtDtEjkusZjzQE3IAix8b0Fxy3Y/edit)|[Zoom](https://zoom.us/j/8203130519)|[Gitter](https://gitter.im/open-telemetry/metrics-specs)
+Specification: Metrics API/SDK|Every Tuesday at 10:00 PT|[Google Doc](https://docs.google.com/document/d/1-bCYkN-DWJq4jw1ybaDZYYmx-WAe6HnwfWbkm8d57v8/edit)|[Zoom](https://zoom.us/j/8203130519)|[Gitter](https://gitter.im/open-telemetry/metrics-specs)
```

Comment:
This will conflict with the Go SIG weekly meeting. Can we choose a different time for this? Possibly alternate on the off times of the data model group? Or earlier, like 8am?

Reply:
Isn't Go meeting on Thursdays?

GoLang: SDK|Every Thursday alternating between 10:00 and 15:00 PDT  Or this doc is outdated

Reply:
> Possibly alternate on the off times of the data model group?

This is a very good idea!

Reply:
I've updated the meeting time based on the suggestion from @MrAlias.

Reply:
> Isn't Go meeting on Thursdays?

@cijothomas looks like I misread this. You are correct it is on Thursday, sorry.

I'm glad we settled on the alternating. That should work well, thanks!